resources/views/common/head.html.twig

```diff
@@ -107,6 +107,11 @@
         crossorigin="anonymous">
     </script>
 
+{# jsPDF / html2canvas #}
+
+    <script src="https://html2canvas.hertzen.com/dist/html2canvas.js" integrity="sha384-vhfRL/EJVW2tOaB2/hQqMb8N29l78w84mJMs8WjKQ5ciRS9XsNeELFxLhVKoNKlQ" crossorigin="anonymous"></script>
```

Comment:
The integrity check is failing for html2canvas, its blocked from downloading.

Reply:
New SRI hash added.